### PR TITLE
Create adarun.yml

### PR DESCRIPTION
**Pull request by cmdada**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [ ] Any software project you are adding was first released more than 4 months ago.
- [ ] Any software project you are adding has working installation instructions.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/adarun.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+website_url: "https://github.com/cmdada/adarun"
+source_code_url: "https://github.com/cmdada/adarun"
+description: "Selfhostable homelab dashboard using client storage and web based configuration ."
+licenses:
+  - GPL-3.0
+platforms:
+  - Javascript
+tags:
+  - Personal Dashboards
+depends_3rdparty: false
```
Comment (kokomo123) [inline software/adarun.yml:11]:
```suggestion
```
No need as this line is already defaulted to false 👍🏻

File: software/adarun.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: "adarun"
+website_url: "https://github.com/cmdada/adarun"
+source_code_url: "https://github.com/cmdada/adarun"
+description: "Selfhostable homelab dashboard using client storage and web based configuration ."
+licenses:
+  - GPL-3.0
+platforms:
```
Comment (Mik77751) [inline software/adarun.yml:7]: Gyvcdmn

File: software/adarun.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  - Javascript
+tags:
+  - Personal Dashboards
```
Comment (Mik77751) [inline software/adarun.yml:10]: 56

File: software/adarun.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+name: "adarun"
+website_url: "https://github.com/cmdada/adarun"
```
Comment (Mik77751) [inline software/adarun.yml:2]: Bff

**Comment by kokomo123** [comment]
Please ignore up above, a spambot ^

**Comment by nodiscc** [comment]
Hi, thanks for your contribution.

Currently, this project doesn't have any tagged releases. Our guidelines require that _Any software project you are adding was first released more than 4 months ago._ This count initiates only after a release has been created to ensure users need not rely on the latest development version to use the project.

We also recommend maintaining a simple [changelog](https://keepachangelog.com) that will help users in tracking software changes, particularly those that are breaking changes or require configuration modifications.

Once a release has been created and the project has matured for at least 4 months, the project may be resubmitted to awesome-selfhosted.

Thanks for understanding, and good luck with this project.

**Pull request closed without merging**